src/styles/Components/Description.sass

```diff
@@ -0,0 +1,106 @@
+.description
+  width: 100%
+  height: 100vh
+  padding: 100px 150px 
+  background-color: #fbfbfb
+  display: flex
+  justify-content: center
+  align-items: flex-start
+
+  &__aside
+    width: 30%
+    margin-right: 40px
+    padding: 40px 0px
+
+    h1
+      width: 50%
+      font-size: 45px
+      padding-bottom: 40px
+
+    &--paragraph
+      color: #aaa
+      font-size: 22px
+      padding-bottom: 40px
+      border-bottom: 1px solid #aaa
+
+  &__wideo
+    display: flex
+    align-items: center
+    padding-top: 40px
+
+    button
+      +btn-reset
+      
+      svg 
+        width: 60px
+        height: 60px
+        fill: #4509dd
+    p 
+      padding-left: 10px
+      font-size: 22px
+      color: #aaa
+        
+  &__cards-wrapper
+    width: 100%
+    height: 100%
+    position: relative
+
+    .card:nth-child(2)
+      position: absolute
+      bottom: -5%
+      left: 5%
+    
+    .card:nth-child(3)
+      position: absolute
+      top: 0
+      right: 15%
+        
+
+@media screen and (max-width: 360px) 
+  
+  .description
+    padding: 80px 20px 
+    height: auto
+    flex-direction: column
+    margin-top: 0px
```

Comment:
adding units to 0 is redundant (except for time)

Reply:
Probably, In first version I had margin-top: 'x' px  above in main description class and then I wanted remove the value for mobile version.